pom.xml

```diff
@@ -111,28 +111,6 @@
                     </environments>
                 </configuration>
             </plugin>
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>exec-maven-plugin</artifactId>
-                <version>3.0.0</version>
-                <executions>
-                    <execution>
-                      <phase>verify</phase>
-                      <goals>
-                        <goal>exec</goal>
-                      </goals>
-                    </execution>
-                </executions>
-                <configuration>
-                    <executable>git</executable>
-                    <arguments>
-                      <argument>diff</argument>
-                      <argument>HEAD~1</argument>
-                      <argument>HEAD</argument>
```

Comment:
It looks like this command isn't working in the action.

````
Run git diff HEAD~1 HEAD > show.patch
fatal: ambiguous argument 'HEAD~1': unknown revision or path not in the working tree.
````

Reply:
Issue seems to be fetch depth needs to be configured so we have an actual depth to check.
https://github.com/actions/checkout#usage
````
    # Number of commits to fetch. 0 indicates all history for all branches and tags.
    # Default: 1
    fetch-depth: ''
````